src/serde_arrow_type.erl

```diff
@@ -21,9 +22,7 @@
     | arrow_float().
 %% Any primitive logical type in Apache Arrow that is supported by `serde_arrow'.
 
--type arrow_bool() ::
-    arrow_true
-    | arrow_false.
+-type arrow_bool() :: arrow_boolean.
```

Comment:
IMO we should just call it `boolean` or `b`. We don't call floats `arrow_floats`, we just use `f` for them.

Reply:
Resolving as addressed in the next PR.